src/Articles.jsx

```diff
@@ -1,14 +1,18 @@
 import {useEffect,useState} from 'react'
+import { useParams } from 'react-router-dom';
 import { getArticles } from './api';
 import {Link} from 'react-router-dom';
 
 const Articles = () => {
     const [articleList, setArticleList] = useState([])
     const [isLoading, setIsLoading] = useState(true)
+    const {topic_name}  = useParams();
```

Comment:
Nice use of params so the topic name appears in the URL and users can bookmark a link to a specific topic 🚀